packages/itwinui-css/src/date-picker/date-picker.scss

```diff
@@ -78,47 +77,41 @@ $iui-date-picker-today-circle-size: 32px;
   text-align: center;
   width: $iui-date-picker-cell-width;
   height: $iui-date-picker-cell-height;
-  border-radius: $iui-border-radius;
+  border-radius: var(--iui-border-radius-1);
   font-variant-numeric: tabular-nums;
 }
 
 @mixin iui-calendar-day-base-hover {
   &:hover {
-    font-weight: $iui-font-weight-semibold;
-    color: var(--iui-color-foreground-primary);
-    background-color: rgba(var(--iui-color-foreground-primary-rgb), var(--iui-opacity-6));
+    font-weight: var(--iui-font-weight-semibold);
+    background-color: var(--iui-color-background-transparent-hover);
```

Comment:
I don't like this change from blue to the gray as our UI are black, gray and white, we are lacking more colors and this one is removing what is left.